services/datamanager/datacapture/data_capture_file.go

```diff
@@ -81,7 +85,8 @@ func ReadFile(f *os.File) (*File, error) {
 
 // NewFile creates a new File with the specified md in the specified directory.
 func NewFile(dir string, md *v1.DataCaptureMetadata) (*File, error) {
-	fileName := filepath.Join(dir, getFileTimestampName()) + InProgressFileExt
+	fileName := FilePathWithReplacedReservedChars(
+		filepath.Join(dir, getFileTimestampName()) + InProgressFileExt)
```

Comment:
Out of curiosity what's the point of the timestamp in the filename? Just to make the file names unique and time-ordered? Will this also be the name of the file in blob storage?

Reply:
Correct, keeps file names unique and time ordered locally on the robot, but completely not used in cloud storage.

Reply:
Okay great, thank you!